webxdc.d.ts

```diff
@@ -105,13 +105,15 @@ interface Webxdc<T> {
    */
   importFiles(filter: {
     /**
-     * mimetypes as in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept#unique_file_type_specifiers
-     */
-    mimeTypes?: string[];
-    /** only show files with these extensions.
+     * Only show files with these extensions.
      * All extensions need to start with a dot and have the format `.ext`. */
     extensions?: string[];
-    /** false by default, whether to allow multiple files to be selected */
+    /**
+     * Mime types as in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept#unique_file_type_specifiers
+     * Specifying a mime type requires to list all typical extensions as well.
```

Comment:
this might be recommended, but is it really required? I have seen examples setting only mimetype "image/*" to allow any image, it is unpractical to list all possible image extensions

Reply:
(ah never mind, I realize you are talking in the context of our future implementation of the import API, not the current reality of the internal usage of the input type=file API)

Reply:
yip, exactly. we want to avoid that implementors have to deal with xdc-devs expecting `text/*` to cover all kind of weird office formats

Reply:
If you look at the link there is no such supported thing as `text/*`, its just image, audio and video

Reply:
it would be same with audio/* or whatever